tests/components/plugwise/test_binary_sensor.py

```
@@ -23,7 +23,8 @@
         ("binary_sensor.opentherm_heating", STATE_ON),
         ("binary_sensor.opentherm_cooling_enabled", STATE_OFF),
         ("binary_sensor.opentherm_compressor_state", STATE_ON),
-    ],
+    ], 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)